### PR TITLE
feature: added signup page

### PR DESCRIPTION
**Pull request by aliamaanansari**
https://neocart.netlify.app/signup/

### CODE REVIEW

**Review by poojakelkar — APPROVED** [review]
looks good to merge for me

File: signup/index.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles/main.css" />
+    <link
```
Comment (tanishq20) [inline signup/index.html:8]: Why have you imported the preview link here?

File: signup/index.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      rel="stylesheet"
+      href="https://deploy-preview-8--keen-meninsky-a05bc6.netlify.app/home.css"
+    />
+    <link
+      rel="shortcut icon"
+      href="../assets/favicon.ico"
+      type="image/x-icon"
+    />
+    <link
```
Comment (tanishq20) [inline signup/index.html:17]: You can import the component CSS link directly in the main CSS file.
By this, you will not have to import this link into all files.

File: signup/index.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+      rel="stylesheet"
+      href="https://keen-meninsky-a05bc6.netlify.app/home.css"
+    />
+    <link
```
Comment (tanishq20) [inline signup/index.html:21]: Please merge component PR into the dev branch and remove this preview link.

File: signup/index.html [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>E-commerce</title>
+  </head>
+  <body>
+    <nav class="navbar">
+      <a href="/" class="navbar-title">Neo Cart</a>
+      <div class="navbar-search-input-container">
+        <div class="input-lefticon">
+          <img src="/assets/images/search-icon.svg" alt="" />
```
Comment (tanishq20) [inline signup/index.html:33]: In `img` tag alt can't be empty.
(Apply this to all)

File: signup/styles.css [hunk 5]
```diff
@@ -0,0 +1,105 @@
+.body {
+  background: #f7f7f7;
+}
+.login-container {
+  background: #ffffff;
```
Comment (tanishq20) [inline signup/styles.css:11]: You can use var() for colors.
(Apply this to all)

File: signup/styles.css [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+  box-shadow: 0px 4px 50px rgba(0, 0, 0, 0.05);
+  border-radius: 10px;
+}
+.login-title {
+  display: flex;
+  margin: 2.5rem;
+  justify-content: center;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
```
Comment (tanishq20) [inline signup/styles.css:20]: `rem`  is preferred.

**Review by tanishq20 — COMMENTED** [review]
Suggested some small changes.

**Pull request merged**